docs/Goguen-clusters-HOWTO.org

```diff
@@ -31,34 +34,39 @@ Then ask devops for ssh config fragment to use for connection.
 
    The default type for Mantis clusters is =mantis=.
 
-3. Use the =gac.sh= script available in =~= to create a new deployment checkout:
-: ./gac.sh new --help
-: ./gac.sh new CLUSTER-NAME [CLUSTER-TYPE]
+3. Use the =gac= CLI available in =~= to create a new deployment checkout:
+: gac new --help
```

Comment:
Is this still relevant? 

Do we still have a "bootstrap gac" at the deployer top-level ?

CC. @deepfire